eng/Subsets.props

```diff
@@ -62,7 +62,8 @@
     <DefaultMonoSubsets Condition="'$(MonoAOTEnableLLVM)' == 'true' and '$(MonoAOTLLVMDir)' == ''">mono.llvm+</DefaultMonoSubsets>
     <DefaultMonoSubsets Condition="'$(TargetOS)' == 'Browser'">$(DefaultMonoSubsets)mono.wasmruntime+</DefaultMonoSubsets>
     <DefaultMonoSubsets Condition="'$(MonoCrossAOTTargetOS)' != ''">$(DefaultMonoSubsets)mono.aotcross+</DefaultMonoSubsets>
-    <DefaultMonoSubsets>$(DefaultMonoSubsets)mono.runtime+mono.corelib+mono.packages+mono.tools+</DefaultMonoSubsets>
+    <DefaultMonoSubsets>$(DefaultMonoSubsets)mono.runtime+mono.corelib+mono.packages+</DefaultMonoSubsets>
+    <DefaultMonoSubsets Condition="'$(TargetArchitecture)' != 's390x' and '$(TargetArchitecture)' != 'ppc64le' and '$(TargetArchitecture)' != 'armv6'">$(DefaultMonoSubsets)mono.tools+</DefaultMonoSubsets>
```

Comment:
OOC is there a doc that shows what architectures/platforms/etc that support CoreCLR and all the ones that dont?

Reply:
Would a property like `NonCoreCLRArchitectures` be helpful if there are more architectures in the future that also dont support CoreCLR?

Reply:
I'm not opposed to a property, but I think we're still pretty much winging it with what belongs there - right now there are two ports, linux-s390x and linux-ppc64le which intersect on a) only has Mono, no CoreCLR b) customer is building using source-build c) customer wants to build a complete and total SDK. I threw armv6 into this conditional too because (a) applies, and both (b) and (c) would apply if anyone was actually shipping binaries.

Reply:
It probably makes sense to use `$(PrimaryRuntimeFlavor)` here.